runtime/doc/options.txt

```diff
@@ -1861,6 +1861,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	"line" and "screenline" cannot be used together.
 
 
+			*'cursormvonyank'* *'cmoy'* *'nocursormvonyank'* *'nocmoy'*
+'cursormvonyank' 'cmoy'	boolean	(default on)
```

Comment:
would this fit in 'jumpoptions'? 

else, please look at existing options to find a more conventional name than 'cursormvonyank'

Reply:
I've wanted this before but for all operations as described on the linked issue#12374 . Having one option that only applies to yank might not be worth it, and better implemented on the user config.

Reply:
yes, this PR also needs to clearly and concisely explain why a plugin can't sufficiently solve this.

Reply:
@muniter What are the other operations that need this? I could only think of yank.

@justinmk [A plugin could certainly solve this](https://github.com/svban/YankAssassin.vim), but it was my understanding that people wanted it as an option.
I'll look into 'jumpoptions', I wasn't aware of it.
I'll try to come up with a better name, any suggestions would be appreciated.

Reply:
> [A plugin could certainly solve this](https://github.com/svban/YankAssassin.vim), but it was my understanding that people wanted it as an option.

if the plugin works well, then we probably want to avoid a new option. 
if the plugin looks small enough that we could paste some form of it in the docs.
if the plugin has annoying problems that can't be fixed without a new option or event, then we can consider a new option/event.

Reply:
Fair enough, the plugin I linked does in fact look small. I haven't tested it myself so I can't speak about how well it works. But if that's satisfactory then I suppose this PR isn't necessary.

Reply:
if you are interested in this feature, would you mind testing the plugin ? :)

Reply:
Alright, I'll give it a shot

Reply:
So I played around with it a bit. It does work pretty well. Can't complain.